pages/web/hosting/htaccess_how_to_protect_wordpress/guide.it-it.md

```diff
@@ -0,0 +1,170 @@
+
+### Impedire la visualizzazione delle directory e delle sottodirectory
+
+Per evitare di permettere a tutti i visitatori del tuo sito di visualizzare il contenuto delle sotto-directory (e accidentalmente dare informazioni ai pirati sui temi o sulle estensioni utilizzate), blocca la visualizzazione del contenuto aggiungendo questa riga nel tuo file **.htaccess**:
+
+```bash
+Options All -Indexes
+```
+
+### Proteggi il tuo file di configurazione
+
+Il tuo file **wp-config.php**, presente alla radice del tuo sito Web, contiene informazioni di configurazione sensibili. Impedisci l'accesso a questo file aggiungendo queste righe nel tuo file **.htaccess**:
+
+```bash
+<Files ~ "^.*\.([Hh][Tt][AaPp])">
+    order allow,deny
+    deny from all
+    satisfy all
+</Files>
+```
+
+Per maggiori informazioni su questo argomento, consulta la nostra guida sulla ["restrizione di accesso via IP tramite il file.htaccess"](https://docs.ovh.com/it/hosting/htaccess_how_to_block_a_specific_ip_address_from_accessing_your_website/).
+
+### Blocca un indirizzo IP
+
+Se hai identificato un indirizzo IP malevolo, inserisci la riga nel tuo file **.htaccess*:
+
+```bash
+<Limit GET POST>
+    order allow,deny deny from xxx.xxx.xxx.xxx
+    allow from all
+</Limit>
+```
+
+In questo esempio `xxx.xxx.xxx.xxx` indica l'indirizzo IP da bloccare.
+
+Per maggiori informazioni su questo argomento, consulta la nostra guida sulla ["restrizione di accesso via IP tramite il file.htaccess"](https://docs.ovh.com/it/hosting/htaccess_how_to_block_a_specific_ip_address_from_accessing_your_website/).
+
+#### Blocca un indirizzo IP dalla directory wp-admin (o nelle altre directory)
+
+La directory **wp-admin** permette di accedere alla tua interfaccia di amministrazione (il metodo funziona anche con le altre directory, ma corrispondono a URL che non portano a un'interfaccia specifica). Per proteggere questa directory, autorizzi in modo specifico l'accesso a uno o più indirizzi IP utilizzando il seguente codice da inserire nel tuo **.htaccess**:
+
+```bash
+<Limit GET POST PUT>
+    order deny,allow deny from all
+    allow from xxx.xxx.xxx.xxx
+    allow from xxx.xxx.xxx.xxx
+</Limit>
+```
+
+### Informazioni importanti da ricordare
+
+- Salva una versione funzionale del tuo file **.htaccess** prima di effettuare qualsiasi operazione.
+- Se le modifiche apportate provocano un errore, sostituisci (tramite il tuo client FTP) il file **.htaccess** online con la versione precedente.
+- Puoi gestire alcuni parametri nel tuo file **wp-config.php**
+- I file **.htaccess** sono particolarmente efficaci per la gestione di URL, reindirizzamenti e la sicurezza del tuo sito Web.
+
+## Spingiti oltre <a name="go-further"></a>
```

Comment:
Please check translation here, does not look like the usual Go Further one

Reply:
DONE